package.json

```diff
@@ -67,6 +68,7 @@
     "eslint-plugin-prettier": "^3.1.4",
     "husky": "^4.3.0",
     "jest": "^26.6.0",
+    "jest-circus": "^26.6.3",
```

Comment:
Do you use the jest-circus runner for something specific?

Reply:
No real reason, just code transplanted from another repository.